src/content/docs/en/user-guide/transfer-tokens.md

```diff
@@ -1,18 +1,21 @@
 ---
 section: gettingStarted
-date: Last Modified
+date: 01.27.2024
```

Comment:
We haven't had any sort of standard around using the date field here afaik, so let me check on this before approving.